tensorboard/webapp/runs/data_source/BUILD

```diff
@@ -24,7 +24,10 @@ tf_ts_library(
     srcs = [
         "runs_backend_types.ts",
```

Comment:
I would like to move the hparam types out of this file but don't want to block this greater refactor on that move.